custom/icds_reports/utils/aggregation_helpers/distributed/agg_awc.py

```diff
@@ -204,6 +205,13 @@ def updates(self):
                 CASE WHEN %(month_end_11yr)s > dob AND %(month_start_15yr)s <= dob AND sex = 'F'
                 THEN 1 ELSE 0 END
             ) as cases_person_adolescent_girls_11_14_all,
+            SUM(CASE WHEN ( (out_of_school or re_out_of_school) AND
```

Comment:
added this script update in different commit to ensure that this join is not causing any issue is existing calculation which is ensured when the test cases passed locally with query change(without the  change in the fixture)